tests/ci/pr_info.py

```diff
@@ -264,16 +264,11 @@ def fetch_changed_files(self):
             response = get_gh_api(
                 diff_url,
                 sleep=RETRY_SLEEP,
+                headers={"Accept": "application/vnd.github.v3.diff"},
             )
             response.raise_for_status()
-            if "commits" in self.event and self.number == 0:
-                diff = response.json()
-
-                if "files" in diff:
-                    self.changed_files = {f["filename"] for f in diff["files"]}
-            else:
-                diff_object = PatchSet(response.text)
-                self.changed_files.update({f.path for f in diff_object})
+            diff_object = PatchSet(response.text)
```

Comment:
Let's remove patchset and use only `compare/ref1...ref2` json. And remove unidiff from https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/worker/prepare-ci-ami.sh#L93C45-L93C52